corehq/apps/mocha/templates/mocha/bootstrap2/base.html

```diff
@@ -3,4 +3,5 @@
 
 {% block core_libraries %}
 {% include "style/includes/bootstrap2/core_libraries.html" %}
+<script src="{% static 'style/lib/bootstrap2/bootstrap.min.js' %}"></script>
```

Comment:
actually that file gets loaded in the `bootstrap2/base.html`, so you may want to make sure that that file isn't loaded twice or something